src/patternfly/demos/Toolbar/examples/Toolbar.md

```diff
@@ -187,30 +187,30 @@ import './Toolbar.css'
           {{#> chip-group-list chip-group-list--attribute=(concat 'aria-labelledby="' toolbar--id '-chip-group-label"')}}
             {{#> chip-group-list-item}}
               {{#> chip}}
-                {{#> chip-text chip-text--attribute=(concat 'id="' chip-group--id 'chip-one"')}}
+                {{#> chip-text chip-text--attribute=(concat 'id="' toolbar--id 'chip-one"')}}
```

Comment:
This works since there is only a single chip group in this toolbar, but we have a couple of examples with a toolbar that has multiple chip groups - https://www.patternfly.org/v4/components/toolbar/html#selected-filters-on-mobile-filters-collapsed-expandable-content-expanded

IMO it would save us some work later to leave these as referencing `chip-group--id`, and also update the `{{#> chp-group-label` and `{{#> chip-group-list` params to reference `chip-group--id`, in case we add more chip groups to these toolbar demos.